### PR TITLE
Feature/gbd cause detail

### PR DESCRIPTION
**Pull request by ctroeger**
I have completed an attempt to describe GBD Cause of Death including input data, modeling process, and where/how to pull results.

I would especially like feedback on where this document should live and the appropriate amount of useful information that I have included.

### CODE REVIEW

**Comment by yongqx** [comment]
~/vivarium_research/docs/source/models/gbd_cause_details.rst, It could be under same repository as cause.rst

File: docs/source/models/gbd2017/causes/causes.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Causes in the GBD are organized in a hierarchy where at every level of that hierarchy
+causes of death or disease are mutually exclusive and collectively exhaustive. 
+
+- All causes
+
+  - Injuries, communicable diseases, non-communicable diseases
+
+    - ... down to the most detailed level
+
+Some causes are only associated with non-fatal outcomes, such as lower back pain.
```
Comment (alibow) [inline docs/source/models/gbd2017/causes/causes.rst]: I think this line was repeated and should be deleted :)

File: docs/source/models/gbd2017/causes/causes.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Measures for causes include:  
+Deaths, incidence, prevalence, YLDs, YLLs, and DALYs
+
```
Comment (alibow) [inline docs/source/models/gbd2017/causes/causes.rst]: Should we also include metrics here? As in counts/rates/percent?

File: docs/source/models/gbd2017/causes/causes.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	
+Modeling causes of death
+------------------------
+
+.. image:: fatal_flowchart.png
+   :width: 600
+
+**Modeling approach** 
+
+Most causes use a centrally-maintained Bayesian ensemble model (CODEm)
```
Comment (alibow) [inline docs/source/models/gbd2017/causes/causes.rst]: "use" seems a little vague here to me. Maybe something like "estimates for most causes are computed using a centrally-maintained...." (if that is factually correct)

File: docs/source/models/gbd2017/causes/causes.rst (hunk 1 above)
Comment (ctroeger) [reply docs/source/models/gbd2017/causes/causes.rst]: Thanks!

File: docs/source/models/gbd2017/causes/causes.rst (hunk 2 above)
Comment (ctroeger) [reply docs/source/models/gbd2017/causes/causes.rst]: I added that suggestion!

File: docs/source/models/gbd2017/causes/causes.rst (hunk 3 above)
Comment (ctroeger) [reply docs/source/models/gbd2017/causes/causes.rst]: I changed the text to "Most causes are estimated using ..."

File: docs/source/models/gbd2017/causes/causes.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+.. image:: fatal_flowchart.png
+   :width: 600
+
+**Modeling approach** 
+
+Most causes are estimated using a centrally-maintained Bayesian ensemble modeling tool (CODEm)
+
+- Bayesian: estimates based on probabilities
+- Ensemble: combines many unique submodels into a final model
+
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/causes.rst]: Paper out today that provides some details on CODEm in a style intended to be understandable:  https://doi.org/10.1093/ije/dyz223

**Pull request merged**